Portfolio/index.html

```diff
@@ -6,8 +6,98 @@
+    <img class="projectimg" src="/public/johnDoe.png">
+    <h3>GET  TO KNOW ME</h3>
+    <p> I am an ambitious and driven engineering student in my final year. I have a strong skill set in HTML, CSS, Javascript, Typescript, React, Sql and Python, and I have my sights set on front-end development and design as a profession. I am also a professional dancer and have strong leadership and management skills. I currently hold a 9.6 CGPA from Medicaps University and am the Operation Head at GDSC. An impressive academic and creative individual, I am sure to make a mark in the tech industry soon.<br><br>
+      <ul>
+     <li> Name: Vanshika Juneja</li>
+     <li> Age: 20</li>
+     <li> Education: Graduating Student</li>
+     <li> Currently in final year.</li>
+     <li> Computer-Science Engineer</li>
+      </ul>
+    </p>
+    <p>
+      <h3>SKILLS</h3>
+        <p><ul>
+          <li><b>Technical Skills:</b></li>
+  <br>
+  <ol>
+          <li>Front end: html css js, ts, jsx , react</li>
+          <li>Core Subjects : oop, cn , dbms, os</li>
+          <li>Others: git, sql, netlify, ms office, canva, figma </li>
+          <li>Tools: VS code, github , slack , discord, replit, notion</li>
+          <li>Programming Languages: C++, Python</li>
+  </ol><br>
+  <li><b> Soft Skiils: </b> </li>
+  <br>
+  <ol>
+  <li>Writing : Content writing, technical writing.
+  </li>
+  <li>Presentation skill</li>
+  <li> Public Speaking</li>
+  <li>Effective Team leading / Team work</li>
+  <li>Creative Thinking</li>
+  <li>Communication skills</li>
+  </ol>
+</ul>  </p></p>
+<p>
+<h3>PROJECTS</h3>
+<ul>
+                  <li>
+                  VogueEco: Build for reducing bad impacts of the textile industry.                    
+                 </li>
+                 <a href="https://www.vogueeco.us/
+                 " class="projecthead"> <button>Explore</button>
+                </a>
+
+                 <img class="projectimg" src="/public/screen1.png">
+                  <li>
+                   Portfolio: Build for Showcasing my Work with Professional Ease.
+                </li>
+                <a href="https://vanshikajuneja.netlify.app/" class="projecthead"><button>Explore</button></a> 
+                  
+                <img class="projectimg" src="/public/screen2.png">
+                  
+                <li>
+                  Finlit:
+                   Build for providing financial education resources and advice.
+                  </li>
+                  <a href="https://finlit.study/" class="projecthead"> <button>Explore</button></a>
+                  <img class="projectimg" src="/public/screen3.png">
+                 
+                  <li>
+                     Book Store: Build as my 1
+                    st project, with basic skills deployed already.
+                    
+                   </li>
+                   <a href=" https://vanshikabook.netlify.app/" class="projecthead"><button>Explore</button></a>
```

Comment:
The <form> tag should have an 'action' attribute specified to indicate where the form data should be submitted.